polars/polars-io/src/ipc/ipc_file.rs

```diff
@@ -77,6 +77,21 @@ pub struct IpcReader<R: MmapBytesReader> {
     metadata: Option<read::FileMetadata>,
 }
 
+fn check_mmap_err(err: PolarsError) -> PolarsResult<()> {
+    match err {
+        PolarsError::ArrowError(e) => match e.as_ref() {
+            arrow::error::Error::NotYetImplemented(s)
+                if s == "mmap can only be done on uncompressed IPC files" =>
+            {
+                eprint!("Could not mmap compressed IPC file, defaulting to normal read. Toggle off 'memory_map' to silence this warning.");
```

Comment:
Would it make sense for this to be eprintln? 

I see that the missing newline is the reason why nushell formatting breaks